src/main/java/com/vmtapp/enterprise/dao/ITicketDao.java

```diff
@@ -1,7 +1,6 @@
 package com.vmtapp.enterprise.dao;
 
 import com.vmtapp.enterprise.dto.Ticket;
-
 import java.util.List;
```

Comment:
https://www.baeldung.com/java-clean-code 
Indentation this just keep it more clean and free up any unused space